jupyterlab/runlab.sh

```diff
@@ -192,34 +217,37 @@ else
 fi
 # The Rubin Lap App plus our environment should get the right hub settings
 # This will need to change for JL 3
-cmd="jupyter-rubinlab \
+cmd="jupyter labhub \
      --ip='*' \
      --port=8888 \
      --no-browser \
      --notebook-dir=${HOME} \
-     --LabApp.shutdown_no_activity_timeout=43200 \
+     --hub-prefix='/nb/hub' \
+     --hub-host='${EXTERNAL_INSTANCE_URL}' \
+     --FileContentsManager.hide_globs=[] \
+     --KernelSpecManager.ensure_native_kernel=False \
+     --LabApp.shutdown_no_activity_timeout=${IDLE_TIMEOUT} \
      --MappingKernelManager.cull_idle_timeout=43200 \
      --MappingKernelManager.cull_connected=True \
-     --MappingKernelManager.default_kernel_name=lsst \
-     --FileContentsManager.hide_globs=[] \
-     --KernelSpecManager.ensure_native_kernel=False"
-#     --SingleUserNotebookApp.hub_api_url=${EXTERNAL_INSTANCE_URL}${JUPYTERHUB_SERVER_PREFIX}
-#     --SingleUserNotebookApp.hub_prefix=${JUPYTERHUB_SERVICE_PREFIX}
-#     --SingleUserNotebookApp.hub_host=${EXTERNAL_INSTANCE_URL}
+     --MappingKernelManager.default_kernel_name=lsst"
+#     --SingleUserNotebookApp.hub_api_url=${EXTERNAL_INSTANCE_URL}${JUPYTERHUB_SERVER_PREFIX} \
```

Comment:
Why are we leaving commented out code in?

Reply:
Ah, good point.  I'll remove it.